packages/turf-center-median/index.d.ts

```diff
@@ -0,0 +1,22 @@
+import { Feature, Point, Coord, FeatureCollection } from '@turf/helpers'
+ 
+/**
+ * http://turfjs.org/docs/#centermedian
+ */
+
+export interface medianProps {
+    medianCandidates: Array<Coord>,
```

Comment:
😛 Did it again, `Coord` is used to define all 3 Point Types.

`Position[]` is the one you are looking for, the following are all equivalent:
- `[number, number][]`
- `Array<[number, number]>`
- `Array<Array<number>>`

Reply:
Rats. I thought for sure I'd changed that. I'm all about that `Position` now.

Reply:
Haha nice! 🤣 

FYI: `Position` is defined here in the GeoJSON specification:

https://tools.ietf.org/html/rfc7946#section-3.1.1

Also defined the same way in the GeoJSON Typescript definition:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/geojson/index.d.ts#L22

Reply:
Nice. That's super useful. I've started learning some of the basics of TypeScript, so I think this kind of typing will come more naturally to me once I have a stronger sense of what a type is and where to find their definitions.